FrontEnd/src/components/Validate/Validate.tsx

```diff
@@ -68,9 +68,18 @@ export function ValidatePure({
       return (
         <>
           <ValidationTable
-            error={true}
             errorMessage={errorsData}
             uploadedData={filesWithErrorsList}
+            headers={[
```

Comment:
These were hardcoded in another spot of the codebase before. At some point these should not be hardcoded, but it's out of scope for this PR.